src/EPR.Payment.Service/Controllers/Payments/OnlinePaymentsController.cs

```diff
@@ -11,15 +11,15 @@ namespace EPR.Payment.Service.Controllers
 {
     [ApiVersion(1)]
     [ApiController]
-    [Route("api/v{version:apiVersion}/payments")]
+    [Route("api/v{version:apiVersion}/onlinepayments")]
```

Comment:
We have hyphenated string for compliance-scheme in the route. I would suggest online-payments for route to be consistent. But not sure it has been agreed with architects? Just suggestion